src/test/java/com/rocketden/main/service/UserServiceTests.java

```diff
@@ -35,4 +41,44 @@ public void createUserSuccess() {
         verify(repository).save(Mockito.any(User.class));
         assertEquals("rocket", response.getNickname());
     }
+
+    @Test
+    public void createUserInvalidNickname() {
+        CreateUserRequest request = new CreateUserRequest();
+        request.setNickname("rocket rocket");
```

Comment:
nit: Add comment on why this nickname is invalid (I can see us changing those details in the future, and if so, we may be confused by this method without an explanatory comment IMO).